### ISSUE TITLE
Incremental training gets slower after each fit call of keras model

### ISSUE THREAD

**Issue by dhatraknilam** [keras-team/keras#13710]
Hello, I am training autoencoder model with real time data by calling fit method of keras model on each occurance of the sample data. To understand the scenario, please refer https://github.com/keras-team/keras/issues/4446, which says we can call fit multiple times.
Raw code for my implementation is:
`for each_occurance_of_the_sample:
model.fit(the_training_sample,the_training_sample)`

As I am using autoencoders input and outputs are the same(mentioning to avoid confusion).

**Comment by dhatraknilam** [comment]
>559
Epoch 1/1
1/1 [==============================] - 233s 233s/step - loss: 0.1518
>560
Epoch 1/1
1/1 [==============================] - 234s 234s/step - loss: 0.1627
>561
Epoch 1/1
1/1 [==============================] - 240s 240s/step - loss: 0.1735

As we can see for 559th sample training time is 233 sec, which is continously increasing as we proceed in the loop. I can not clear the session as I want to retain the graph weights.

**Comment by dhatraknilam** [comment]
I found out the issue, I was creating a new object of the model in every iteration of the consumer loop, which was eating memory as that many instances of the model were allocating memory at the same time, which resulted in slower learning.
